integration-tests/reactive-messaging-kafka/src/test/java/io/quarkus/it/kafka/KafkaConnectorTest.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.it.kafka;
+
+import static io.restassured.RestAssured.get;
+import static org.awaitility.Awaitility.await;
+
+import java.util.List;
+
+import org.jboss.logging.Logger;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.common.mapper.TypeRef;
+
+@QuarkusTest
+@QuarkusTestResource(KafkaTestResource.class)
+public class KafkaConnectorTest {
```

Comment:
Could you add the associated native tests?

```java
package io.quarkus.it.kafka;

import io.quarkus.test.junit.NativeImageTest;

@NativeImageTest
public class KafkaConnectorIT extends KafkaConnectorTest {

}
```

Reply:
Done